### PR TITLE
Buildsystem: allows to not use a requirement file

### PR DESCRIPTION
**Pull request by dzen**
following the previous issue : https://github.com/spotify/dh-virtualenv/issues/182

One can now disable the required requirement file

### CODE REVIEW

File: lib/Debian/Debhelper/Buildsystem/dh_virtualenv.pm [hunk 1]
```diff
@@ -38,9 +38,9 @@ sub get_install_root {
 sub get_venv_builddir {
     my $this = shift;
     my $builddir = $this->get_builddir();
-    my $sourcepackage = $this->sourcepackage();
+    my $virtualenv_name = $ENV{DH_VIRTUALENV_INSTALL_SUFFIX} || $this->sourcepackage();
```
Comment (nailor) [inline lib/Debian/Debhelper/Buildsystem/dh_virtualenv.pm]: Maybe you need to rebase on master?

Comment (nailor) [inline lib/Debian/Debhelper/Buildsystem/dh_virtualenv.pm]: Ah, scratch that, just realised I hadn't merged the previous one and indeed this PR fixes a bug in that, nice catch.

**Comment by nailor** [comment]
I merged your previous change, so please rebase this on top of master.

Now another question, see line 130 in the file (unfortunately I can't comment there). This should probably modify that line to take pip args, and then just skip requirements installation earlier.

One more question: instead adding a feature like this, you could just have empty requirements.txt, right? I know it's sub-par but I'm trying to figure out how useful this feature is in general, as there are a lot of reasons why one should use the requirements file approach.

**Comment by jhermann** [comment]
You could even build in a staging area (which is something I advise anyway), and add a touch cmd to your `rules`, leaving your project dir pristine.

**Comment by dzen** [comment]
Yes, but it sounds more like a hack, and changes the behaviour from using dh_virtualenv CLI.

Since there is multiple way to do a package using dh_virtualenv, should we provide the same features ?

**Comment by dzen** [comment]
well. I repushed the branch, the code is still there ..

**Comment by dzen** [comment]
It's up to you :) I could clearly create a requirement file with ```-e .``` in it

**Pull request closed without merging**